101-webapp-including-insights/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "webappName": {
+      "value": "GEN-UNIQUE"
+    },
+    "appInsightsName": {
+      "value": "GEN-UNIQUE"
+    },
+    "hostingPlanName": {
+      "value": "GEN-UNIQUE"
+    },
+    "owner": {
```

Comment:
remove this since you'll have a defaultValue

Reply:
done